create-release/action.yml

```diff
@@ -18,7 +18,11 @@ inputs:
   ref:
     description: 'The branch or SHA for the release (defaults to main)'
     required: false
-    default: ${{ github.ref }}
+  skip_release:
+    description: 'Skip creating the release and only generate the changelog'
+    required: false
+    default: 'false'
```

Comment:
Instead of this, I would prefer you create a whole new action (like `generate-changelog/action.yml`, pull all of the changelog generating bits into that and reuse here.

Reply:
I can get started on that. Thanks for the review